### PR TITLE
Allow unauthenticated users to view /static/style.css

### PR DESCRIPTION
**Pull request by opsyne**
The /lock route uses the /static/style.css file, but this file is only visible
to authenticated users, leading to an unstyled login screen

### CODE REVIEW

File: misc/handlers.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 
 func InitHandlers(rtr *mux.Router) {
 	rtr.HandleFunc("/robots.txt", handlerRobotsTxt)
-	rtr.HandleFunc("/static/style.css", handlerStyle)
+	rtr.HandleFunc("/static/style.css", HandlerStyle)
 	rtr.PathPrefix("/static/").
 		Handler(http.StripPrefix("/static/", http.FileServer(http.FS(static.FS))))
```
Comment (handlerug) [inline misc/handlers.go:28]: Also you probably should move this stuff to the auth router as well, plus the favicon route. People may depend on static assets in their custom styles, and a favicon is generally not really private, given that we leak header links on auth fail already.

**Review by handlerug — CHANGES_REQUESTED** [review]
Good catch! Seems like it got broken somewhere after the 1.9 release. I've left two comments, take a look if you haven't thought of them.

File: misc/handlers.go (hunk 1 above)
Comment (opsyne) [reply misc/handlers.go]: I think I understand what you mean. I moved the routes to the auth router, but left the handler functions in misc.

File: misc/handlers.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ import (
 
 func InitHandlers(rtr *mux.Router) {
 	rtr.HandleFunc("/robots.txt", handlerRobotsTxt)
-	rtr.HandleFunc("/static/style.css", handlerStyle)
+	rtr.HandleFunc("/static/style.css", HandlerStyle)
```
[the remark this answers is not in the document]
Comment (opsyne) [reply misc/handlers.go]: Makes sense

**Comment by handlerug** [comment]
Thanks!

**Comment by opsyne** [comment]
> Thanks!

No problem!

**Pull request merged**